### PR TITLE
Fix horizontal scrolling

### PR DESCRIPTION
**Pull request by jcshott**
Update react-data-grid to version v5x (which appears to fix no horizontal scrolling)

there are deprecation warnings that show up and this solves it: https://github.com/adazzle/react-data-grid/issues/1403#issuecomment-499631101

I didn't want to upgrade all the way to v7x (which fixed the warnings) as it doesn't seem stable and could open a pandora's box of problems. one version upgrade seems to be ok.

bp-scroll

### CODE REVIEW

File: frontend/src/components/RowRenderer/RowRenderer.jsx [hunk 1]
```diff
@@ -8,17 +8,13 @@ export default class RowRenderer extends React.Component {
     idx: PropTypes.number.isRequired
   };
 
-  setScrollLeft = (scrollBy) => {
```
Comment (jcshott) [inline frontend/src/components/RowRenderer/RowRenderer.jsx:11]: this didn't appear to be used and was just confusing.

**Review by JonathanAquino-NextRoll — APPROVED** [review]
Nice!

**Comment by JonathanAquino-NextRoll** [comment]
Would you be able to link to the react-data-grid PR that fixes horizontal scrolling or a note in the version history on this, if you have it handy?

**Comment by jcshott** [comment]
I'll see if I can find it - after a lot of searching, I just tried updating to see if it worked - figuring a 4 -> 5 would minimize other potential issues (since we don't really use a lot of features) and it seemed to fix it. so I just ran with it :) but I might be able to figure out what was fixed and link here.

**Pull request merged**